packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -30,56 +22,51 @@ export function Service({
 }) {
   const { city, stateCode, country } = location;
   const locationProvided = Boolean(city && stateCode && country);
-  const isPocket = name === 'Pocket';
-  const isMonitor = name === 'Firefox Monitor';
-  const isLockwise = name === 'Firefox Lockwise';
-  const isSync = name === 'Firefox Sync';
-  const isFPN = name === 'Firefox Private Network';
+  let serviceLink, iconName;
 
-  const renderLink = isPocket || isMonitor || isLockwise || isSync || isFPN;
-  let serviceLink = '';
-  let iconSvg = <WebIcon data-testid="web-icon" />;
-
-  if (renderLink) {
-    if (isPocket) {
+  switch (name) {
+    case 'Pocket':
       serviceLink = 'https://www.mozilla.org/en-US/firefox/pocket/';
-      iconSvg = <PocketIcon data-testid="pocket-icon" />;
-    } else if (isMonitor) {
+      iconName = 'pocket';
+      break;
+    case 'Firefox Monitor':
       serviceLink = 'https://monitor.firefox.com/';
-      iconSvg = <MonitorIcon data-testid="monitor-icon" />;
-    } else if (isLockwise) {
+      iconName = 'monitor';
+      break;
+    case 'Firefox Lockwise':
       serviceLink = 'https://www.mozilla.org/en-US/firefox/lockwise/';
-      iconSvg = <LockwiseIcon data-testid="lockwise-icon" />;
-    } else if (isSync) {
-      serviceLink =
-        'https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer';
-      iconSvg = <SyncIcon data-testid="sync-icon" />;
-    } else if (isFPN) {
+      iconName = 'lockwise';
+      break;
+    case 'Firefox Private Network':
       serviceLink = 'https://vpn.mozilla.com/';
-      iconSvg = <FPNIcon data-testid="fpn-icon" />;
-    }
-  } else {
-    const lowerName = name.toLowerCase();
-    if (deviceType === 'mobile') {
-      iconSvg = <MobileIcon data-testid="mobile-icon" />;
-    }
-    if (lowerName.includes('ipad')) {
-      iconSvg = <TabletIcon data-testid="tablet-icon" />;
-    } else if (deviceType === 'desktop') {
-      iconSvg = <DesktopIcon data-testid="desktop-icon" />;
-    }
+      iconName = 'fpn';
+      break;
+    case 'Firefox Sync':
+      serviceLink = 'https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer';
+      iconName = 'sync';
+      break;
```

Comment:
We could use a dictionary here instead.  The names for the keys and the values would be of a type `{serviceLink: someUrlTypeThatIsAnAliasOfString, iconName: string}`.  Then we'd map into it for the link and icon.  Just a code style thing; the switch here works.